### PR TITLE
ZBUG-1460:- Fixed Cross-Site Scripting vulnerability in Zimbra web mail with account name.

### PR DESCRIPTION
**Pull request by madhavdhuppe**

### CODE REVIEW

File: WebRoot/js/zimbraMail/share/model/ZmZimbraAccount.js [hunk 1]
```diff
@@ -110,7 +110,7 @@ function(name) {
 	var identity = this.getIdentity();
 	// TODO: If no identity and name is set, should create one!
 	if (!identity) return;
-	identity.name = name;
+	identity.name = AjxStringUtil.htmlEncode(name);
```
Comment (silentsakky) [inline WebRoot/js/zimbraMail/share/model/ZmZimbraAccount.js:113]: we should do html encoding only in the view layer, data should never be modified

File: WebRoot/js/zimbraMail/share/model/ZmZimbraAccount.js [hunk 2]
```diff
@@ -123,7 +123,7 @@ function() {
 	var identity = this.getIdentity();
 	var name = (!identity)
 		? this.settings.get(ZmSetting.DISPLAY_NAME)
-		: identity.name;
+		: AjxStringUtil.htmlDecode(identity.name);
```
Comment (silentsakky) [inline WebRoot/js/zimbraMail/share/model/ZmZimbraAccount.js:126]: htmlDecode should not be done by us, it's responsibility of browser

File: WebRoot/js/zimbraMail/share/model/ZmZimbraAccount.js (hunk 1 above)
Comment (madhavdhuppe) [reply WebRoot/js/zimbraMail/share/model/ZmZimbraAccount.js:113]: but i think the the svg onload is getting  executed while sending requests

File: WebRoot/js/zimbraMail/share/model/ZmZimbraAccount.js (hunk 2 above)
Comment (madhavdhuppe) [reply WebRoot/js/zimbraMail/share/model/ZmZimbraAccount.js:126]: I have some places that we are using AjxStringUtil.htmlDecode()

File: WebRoot/js/zimbraMail/share/model/ZmZimbraAccount.js (hunk 1 above)
Comment (madhavdhuppe) [reply WebRoot/js/zimbraMail/share/model/ZmZimbraAccount.js:113]: updated

**Pull request merged**